src/main/java/com/redhat/composer/components/application/ContentRetrieverConfig.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.composer.config.application;
+package com.redhat.composer.components.application;
```

Comment:
For the other classes, I agree that the `com.redhat.composer.components` package makes more sense. For this configuration class, I believe `com.redhat.composer.config` continues to be a better package for it. Since the other classes are moving, you could drop the `application` from the package. Just a thought.